File: 02-Processos-e-Threads/03-Processo-JVM/java/Proc.java
public class Proc {
	public static void main(String[] args) {
		final ProcessHandle currProc = ProcessHandle.current();
		System.out.println(":: RUNNING WITH PID " + currProc.pid() + " ::");
		System.out.println("Check Task Manager > Details > java.exe");
		System.out.println("(press ENTER to stop)");
		System.console().readLine();
		System.out.println(":: DONE ::");
	}
}

File: 02-Processos-e-Threads/03-Processo-JVM/kotlin/proc.kt
fun main() {
	val currProc = ProcessHandle.current()
	println(":: RUNNING WITH PID ${currProc.pid()} ::")
	println("Check Task Manager > Details > java.exe")
	println("(press ENTER to stop)")
	System.console().readLine()
	
	println(":: DONE ::")
}
